src/Components/CreateImageWizard/steps/Review/ReviewStepTextLists.tsx

```diff
@@ -817,7 +817,7 @@ export const UsersList = () => {
             Password
           </TextListItem>
           <TextListItem component={TextListItemVariants.dd}>
-            {userPassword ? userPassword : 'None'}
+            {userPassword ? '●'.repeat(8) : 'None'}
```

Comment:
I wonder if we should allow to show the password on the review step with a similar toggle we use on the Users step (the eye icon).

Or maybe not render the password at all? I guess that would be a bit weird. It's just that rendering a hidden password does not seem super useful. It gives the information that _a_ password was selected, but when the user want's to check what password that was they have to return to the user step.

Reply:
Ah, gotcha. Alright, let's get this in and if we need a "show password" toggle we can add it in a follow up.